examples/kubernetes/manifests/mariadb-secrets.yml

```diff
@@ -0,0 +1,11 @@
+# NOTE: this file should be hidden somehow, or recreated only from the CLI. This
+# has been done this way to illustrate better the example.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mariadb-secrets
```

Comment:
On second thought, it might be nicer to have the example put everything in a `portus` namespace, since that's a cleaner way to run it anyway